mam/examples/recv.c

```diff
@@ -37,6 +38,11 @@ int main(int ac, char **av) {
   receive_bundle(av[1], atoi(av[2]), (tryte_t *)av[3], bundle);
 
   mam_psk_t_set_add(&api.psks, &psk);
+  if (ac == 5) {
+    trytes_to_trits((tryte_t *)av[4], chid.pk,
+                    MAM_CHANNEL_ID_SIZE / NUMBER_OF_TRITS_IN_A_TRYTE);
+    mam_pk_t_set_add(&api.trusted_channel_ids, &chid);
```

Comment:
Can you provide 2 api functions that do these two operations ?
They are actually #796  and #795

Reply:
yes